standard/types.md

```diff
@@ -722,19 +722,14 @@ An *unmanaged_type* is any type that isn’t a *reference_type*, a *type_paramet
 
 Any expression of a reference type can either refer to an object, or be the value `null`, which indicates "no reference". The terms "nullable reference type" and "non-nullable reference type" refer to annotations that declare whether an expression is intended to permit null values or not. A compiler can provide diagnostics when a expression is not used according to that intent.
 
-There are three forms of nullability for reference types:
+There are two forms of nullability for reference types:
 
-- *null-oblivious*: A *null-oblivious* type can be assigned `null`. Its default null state is *not-null*.
 - *nullable*: A *nullable-reference-type* can be assigned `null`. Its default null state is *maybe-null*.
 - *non-nullable*" A *non-nullable reference* should not be assigned a `null` value. Its default null state is *not-null*.
 
 Unlike with nullable value types, where value types `V` and `V?` denote different types, given a reference type `R`, the notations `R` and `R?` denote the same underlying type; the difference in their notations indicates, at compile time, only the intent of their usage, and allows for static flow analysis. An identity conversion exists among a nullable reference type, its corresponding non-nullable reference type, and its corresponding null-oblivious reference type (§10.2.2).
 
-> *Note:* By definition, a reference type is nullable; that is, a variable of that type can either contain a reference to an object or be the value `null`, which indicates “no reference.” The disabled nullable context matches the previous standard behavior for reference types. All reference types in these programs are *null-oblivious* reference types. *end note*
-
-### §Null-oblivious-reference-types Null oblivious reference types
```

Comment:
I think we will still need some kind of section defining what null-oblivious means - it's just that instead of referring to a type, it'll be to the state of an expression. (I *think*, anyway...)

Reply:
I think I finessed this is in the latest commit. Let me know what you think.